brain/compartmentReportView.cpp

```diff
@@ -99,6 +99,36 @@ std::future<brion::Frames> CompartmentReportView::load(double start, double end)
     return _impl->report->loadFrames(start, end);
 }
 
+std::future<brion::Frames> CompartmentReportView::load(double start, double end,
```

Comment:
Do you have an estimate of the performance comparison between loadAll and load all with stride 2*timestep?